apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html

```diff
@@ -146,7 +146,7 @@ <h3 class="font-16 d-flex" [checkPermission]="['MENU', 'MANAGE']" dropdownToggle
                         class="d-inline-flex align-items-center cursor-pointer">
                         Menu
                         <i class="icon-down-new mr-l05"></i> </span>
-                    <span class="text-gray cursor-pointer" (click)="showNavigationModal()">
+                    <span class="text-gray cursor-pointer" [routerLink]="['/pages/giddh-all-items']">
```

Comment:
Please delete `showNavigationModal` method from `.ts` file if it is not used anywhere